backend/oasst_backend/tree_manager.py

```diff
@@ -522,6 +522,9 @@ async def handle_interaction(self, interaction: protocol_schema.AnyInteraction)
                     )
```

Comment:
let's move the message size check also to `store_text_reply()` in `PromptRepository` ..